### PR TITLE
:mortar_board: Topic overview --- Example section

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Write the example part with n queens as the example 

### Testing

:+1: built local, looks good 

### Additional Notes

The reps discussed in the example are both kinda' terrible, but going into details on the rep a topic of discussion for the next topic.

### CODE REVIEW

**Review by twentylemon — APPROVED** [review]
What's here is fine. I do think something like travelling salesman problem is easier to understand though, as there is no obvious objective truth for the optimiation variant. Gradient descent becomes more noticable and understandable.

**Comment by jameshughes89** [comment]
> What's here is fine. I do think something like travelling salesman problem is easier to understand though, as there is no obvious objective truth for the optimiation variant. Gradient descent becomes more noticable and understandable.

Hmmm... The reason I went with n queens was because the next topic will use it so it would segue nice. That said, I think you might be right in that TSP will be must easier to follow.

File: site/topics/overview/overview.rst [hunk 1]
```diff
@@ -333,87 +334,110 @@ Example
...
-
-    * Instead of looking for a valid board configuration, minimize the number of conflicts
-    * This provides a gradient
+* The phenotype is the actual Hamiltonian cycle
+* The genotype is encoding a Hamiltonian cycle as a permutation
 
+* The fitness would be the total length of the Hamiltonian cycle
+* Given a chromosome, sum up the distances between the cities in order
 
-* An appropriate fitness function for this problem would be to count the number of conflicting queens
```
Comment (twentylemon) [inline site/topics/overview/overview.rst]: I'd still be explicit about the distance being the gradient here. It is more obvious than n-queens, but there's no need to force students to read between the lines for an introductory example.

File: site/topics/overview/tsp_example.png
Comment (twentylemon) [inline site/topics/overview/tsp_example.png:1]: Pff, lame tsp. Here's one I did from school

image

**Review by twentylemon — APPROVED** [review]

File: site/topics/overview/tsp_example.png
Comment (jameshughes89) [reply site/topics/overview/tsp_example.png:1]: 🦓

**Pull request merged**